apps/stats/stats-1.py

```diff
@@ -89,19 +89,12 @@ def __init__( self ) :
 				),
 
 				IECore.IntParameter(
-					name = "mostFrequentlyHashed",
```

Comment:
Did these mostFrequentlyHashed and mostFrequentlyComputed ever make it into a tag or are they just on master? I'm not sure I'd consider it an incompatibility if they haven't got past master yet.